coops-ui/src/components/AnnualReport/Certify.vue

```diff
@@ -57,38 +57,39 @@ export default class Certify extends Vue {
    * Lifecycle callback to always give the parent a "valid" event for its property values.
    */
   private created (): void {
-    this.emitValid(this.trimmedCertifiedBy && this.isCertified)
+    this.emitValid(Boolean(this.trimmedCertifiedBy && this.isCertified))
```

Comment:
I like it. Another option for type coercion to a `Boolean` would be `!!(this.trimmedCertifiedBy && this.isCertified)`. I think it's debatable which is more clear or concise though. Just a thought!

Reply:
Thanks.